packages/zoe/src/contractSupport/zoeHelpers.js

```diff
@@ -207,6 +207,12 @@ export const depositToSeat = async (zcf, recipientSeat, amounts, payments) => {
   const invitation = zcf.makeInvitation(
     reallocateAfterDeposit,
     'temporary seat for deposit',
+    undefined,
+    harden({
+      give: M.any(),
```

Comment:
doesn't it have to be an `AmountKeywordRecord`?
```suggestion
      give: AmountKeywordRecordShape,
```